nemo_skills/inference/generate.py

```diff
@@ -67,6 +67,7 @@ class GenerateSolutionsConfig:
     max_samples: int = -1  # If > 0, will stop after generating this many samples. Useful for debugging
     skip_filled: bool = False  # If True, will skip the generations that are already in the output file
 
+    max_concurrent_requests: int = 128  # Maximum number of concurrent requests to the server for the async loop
```

Comment:
```suggestion
    max_concurrent_requests: int = 1024  # Maximum number of concurrent requests to the server for the async loop
```
I think 1024 is a good default to utilize most engines. 128 is too small